api/Trace/Events.php

```diff
@@ -15,6 +15,5 @@ interface Events extends \IteratorAggregate, \Countable
      */
     public function addEvent(string $name, ?Attributes $attributes = null, ?int $timestamp = null): Events;
 
-    public function count(): int;
```

Comment:
This method is already added since the interface extends `Countable` and was causing issues with duplicate code suggestions and psalm `pure` methods i been leveraging.